Makefile

```diff
@@ -4,18 +4,13 @@ imports = \
 xcodeproj:
 	PF_DEVELOP=1 swift run xcodegen
 
-linux-main:
-	sourcery \
-		--sources ./Tests/ \
-		--templates ./.sourcery-templates/ \
-		--output ./Tests/ \
-		--args testimports='$(imports)' \
-		&& mv ./Tests/LinuxMain.generated.swift ./Tests/LinuxMain.swift
-
-test-linux: linux-main
+test-linux: 
```

Comment:
This should still depend on `linux-main`, right?
```suggestion
test-linux: linux-main
```

Reply:
Oh, or maybe not...it still runs on Linux I guess, no?

Reply:
Right. Maybe we need the difference of running Linux tests from macOS versus Linux, but I think this is ok